modules/exploits/multi/http/solr_velocity_rce.rb

```diff
@@ -0,0 +1,354 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+###
+#
+# This exploit sample shows how an exploit module could be written to exploit
+# a bug in an arbitrary TCP server.
+#
+###
+require 'json'
+require 'msf/core/exploit/powershell'
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  #
+  # This exploit affects TCP servers, so we use the TCP client mixin.
+  # See ./documentation/samples/vulnapps/testsrv/testsrv.c for building the
+  # vulnerable target program.
+  #
```

Comment:
Remove

Reply:
Fixed in commit 046d8cbedc659abf7ec2edeb922b3df71c046a22.